### PR TITLE
feat(docs): describe logging from scripts

### PR DESCRIPTION
**Pull request by yuri1969**
This addition documents how to emit standard log messages from script tasks.

### CODE REVIEW

File: content/docs/04.workflow-components/01.tasks/02.scripts/08.logging.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+Node.js follows the same syntax for sending logs as in Python. Here is an example:
+
+You need to install the [npm package](https://www.npmjs.com/package/@kestra-io/libs), that can be done with a `beforeCommands`:
+
+```yaml
+beforeCommands:
+ - npm i @kestra-io/libs
+```
+
+The just require or import the package:
```
Comment (Skraye) [inline content/docs/04.workflow-components/01.tasks/02.scripts/08.logging.md:72]: Sentence not clear

**Review by Skraye — CHANGES_REQUESTED** [review]
One small issue with a sentence, however very good!

File: content/docs/04.workflow-components/01.tasks/02.scripts/08.logging.md (hunk 1 above)
Comment (anna-geller) [inline content/docs/04.workflow-components/01.tasks/02.scripts/08.logging.md:72]:
```suggestion
Then, simply use the `require` function to import the Kestra package and emit logs:
```

**Review by anna-geller — APPROVED** [review]
Thanks so much! <3

**Pull request merged**